jans-cedarling/bindings/cedarling-java/README.md

```
@@ -67,25 +67,93 @@ To use Cedarling Java bindings in Java Maven Project add following `repository`
+
+1. Upload [bootstrap.json](./docs/bootstrap.json) and [update_token_script.cjar](./docs/update_token_script.cjar)  at `/opt/jans/jetty/jans-auth/custom/static` location of the auth server.
+2. Upload the generated `cedarling-java-{version}-distribution.jar` at `/opt/jans/jetty/jans-auth/custom/libs` location of the auth server. Rather than building the `cedarling-java-{version}.jar` from source code, you can directly download the latest version of the jar from the [Maven repository](https://maven.jans.io/maven/io/jans/cedarling-java/).
+3. The following java [Update Token](./docs/sample_cedarling_update_token.java) script has been created for calling Cedarling authorization. Enable the script with following Custom Properties:
+
+    | Key                  | Values                          |
+    | -------------------- | ------------------------------- |
+    | BOOTSTRAP_JSON_PATH  | ./custom/static/bootstrap.json  |
 
 4. Map the script with client used to perform authentication.
 
 ![](./docs/mapping_post_authn_script_with_client.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)